dspback/main.py

```diff
@@ -32,6 +34,11 @@
 app.include_router(submissions.router, prefix="/api", tags=["Submissions"])
 
 
+@app.exception_handler(RepositoryException)
+async def http_exception_handler(request, exc):
+    return PlainTextResponse(f"Repository exception response[{str(exc.detail)}]", status_code=exc.status_code)
```

Comment:
@Maurier - here is the message wrapping the response. The status code will be whatever the repository responds with.